<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSH_MinHash</class>
 <widget class="QWidget" name="LSH_MinHash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LSH_MinHash</string>
  </property>
  <property name="whatsThis">
   <string>LSH MinHash Program</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>415</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="modeLabel">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileNamesLabel">
        <property name="text">
         <string>Filenames</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fileTypeLabel">
        <property name="text">
         <string>File type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="numberOfSequencesLabel">
        <property name="text">
         <string>Number of Sequences</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="sequenceMinLengthLabel">
        <property name="text">
         <string>Sequence min-length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="subSequenceWidthLabel">
        <property name="text">
         <string>Sub-sequence width</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="hashTypeLabel">
        <property name="text">
         <string>Hash type</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="numberOfBucketsLabel">
        <property name="text">
         <string>Number of buckets</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="toleranceLabel">
        <property name="text">
         <string>Max. No. of Incorrect Letters</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="numberOfHashesLabel">
        <property name="text">
         <string>Number of hashes</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="minSignaturesSimilarityRatioLabel">
        <property name="text">
         <string>Min-signatures similarity ratio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="modeLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="fileNamesLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="fileTypeLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="numberOfSequencesLineEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="sequenceMinLengthLineEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="subSequenceWidthLineEdit"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="hashTypeLineEdit"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="numberOfBucketsLineEdit"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="toleranceLineEdit"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="numberOfHashesLineEdit"/>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="minSignaturesSimilarityRatioLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="runPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openParametersFilePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open JSON parameters file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Open Parameters File</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
